scripts/docker/docker.lib.sh

```diff
@@ -25,7 +25,6 @@ function docker-build() {
   _create-effective-dockerfile
   _create-effective-version
   docker build \
-    --progress=plain \
```

Comment:
This file comes from the repository template, therefore we probably don't want to change it.